napari/layers/tracks/_track_utils.py

```diff
@@ -365,22 +361,24 @@ def ndim(self) -> int:
         return self.data.shape[1] - 1
 
     @property
-    def max_time(self) -> int:
+    def max_time(self) -> Optional[int]:
         """Determine the maximum timestamp of the dataset"""
-        return int(np.max(self.track_times))
+        if self.track_times is not None:
+            return int(np.max(self.track_times))
+        return None
 
     @property
-    def track_vertices(self) -> np.ndarray:
+    def track_vertices(self) -> Optional[np.ndarray]:
         """return the track vertices"""
         return self._track_vertices
 
     @property
-    def track_connex(self) -> np.ndarray:
+    def track_connex(self) -> Optional[np.ndarray]:
         """vertex connections for drawing track lines"""
         return self._track_connex
 
     @property
-    def graph_vertices(self) -> np.ndarray:
+    def graph_vertices(self) -> Optional[np.ndarray]:
         """return the graph vertices"""
         return self._graph_vertices
```

Comment:
Same questions here, maybe later find out if empty arrays make sens instead of None.